node-runtime/src/http-server.ts

```diff
@@ -354,6 +355,7 @@ export class SdkgenHttpServer<ExtraContextT = {}> {
                                             type instanceof BigIntPrimitiveType ||
                                             type instanceof CpfPrimitiveType ||
                                             type instanceof CnpjPrimitiveType ||
+                                            type instanceof HtmlPrimitiveType ||
```

Comment:
This should only have types that are encodable as path, query or header. I don't think html qualifies.